### PR TITLE
BZ-1722739: Adding the new mandatory second parameter

### PR DESCRIPTION
**Pull request by bergerhoffer**
This PR is for 4.1 only.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@hexfusion Can you and Suresh take a look at this please?

**Review by retroflexer — APPROVED** [review]
Looks good to me.
/lgtm

**Review by hexfusion — APPROVED** [review]

File: modules/dr-recover-lost-control-plane-hosts.adoc [hunk 1]
```diff
@@ -261,7 +261,7 @@ ETCD_INITIAL_ADVERTISE_PEER_URLS="https://etcd-1.clustername.devcluster.openshif
 ETCD_INITIAL_CLUSTER_STATE="existing"
 Starting etcd..
 ----
-<1> The IP address of the original master, where the signer server is running.
+<1> Pass in two parameters: the IP address of the original master where the signer server is running, and the etcd name of the new member.
```
Comment (kalexand-rh) [inline modules/dr-recover-lost-control-plane-hosts.adoc]: I'd say "Specify both the IP .... and the etcd..."

**Comment by bergerhoffer** [comment]
Closing the enterprise-4.2 cherry pick - this is for 4.1 only.

**Pull request merged**